geometry_msgs/Pose goal
float64[] start
string[] names

float64 egraph_eps
float64 final_egraph_eps
float64 dec_egraph_eps

float64 initial_eps
float64 final_eps
float64 dec_eps

bool feedback_paths
bool save_egraph
bool use_egraph

---

trajectory_msgs/JointTrajectory path
string[] stat_names
float64[] stat_values
